circle.yml

```diff
@@ -482,6 +482,19 @@ jobs:
           min_time: "0.01"
       - spectest
 
+  x86-linux:
+    docker:
+      - image: ethereum/cpp-build-env:15-gcc-10-multilib
+    steps:
+      - install_testfloat
+      - checkout
+      - build:
+          build_type: Debug
+          cmake_options: -DCMAKE_TOOLCHAIN_FILE=~/project/cmake/toolchains/32bit.cmake
+      - test
+      # TODO: Disabled, because fixes to NaN payload handling in the tool are needed.
```

Comment:
As long as you are 100% sure it is only a tooling fix (and no fix/change needed in the interpreter), I'm fine merging this.

Reply:
I'm not sure, but this is the first thing to fix.